### PR TITLE
CNFT.IO

### PR DESCRIPTION
**Pull request by itsmestale**
* Name: CNFT.IO
* Description: CNFT.IO Is the home to the Cardano NFT community, and now the home to the first NFT marketplace on cardano.
* Link:   https://cnft.io
* Tags:
  * `analytics`
  * `nft`

#### Description of the change

Added cnft.jpg and showcase data.

### CODE REVIEW

File: src/data/showcases.js [hunk 1]
```diff
@@ -665,6 +665,15 @@ const Showcases = [
     source: null,
     tags: ["nft"],
   },
+  {
+  title: "CNFT.IO - NFT Marketplace on Cardano",
+    description:
+      "Mint, Launch, Buy and Sell Cardano NFTS on CNFT.IO. The first automated marketplace on Cardano.",
```
Comment (tweakch) [inline src/data/showcases.js:671]: Hi @itsmestale 
Please read the [About description section](https://github.com/cardano-foundation/developer-portal/blob/staging/CONTRIBUTING.md#a-word-about-project-descriptions) and change your description accordingly so we can approve your PR

**Review by tweakch — CHANGES_REQUESTED** [review]

File: src/data/showcases.js (hunk 1 above)
Comment (itsmestale) [reply src/data/showcases.js:671]: gotcha will do

**Review by tweakch — APPROVED** [review]

**Review by katomm — COMMENTED** [review]
Hi, can you briefly elaborate on why the analytics tag was set? It doesn't seem to fit into this category. I'd suggest "token" and "nft".

**Comment by itsmestale** [comment]
> Hi, can you briefly elaborate on why the analytics tag was set? It doesn't seem to fit into this category. I'd suggest "token" and "nft".

We just launched a few days ago, one of things I am working on atm are data and analytics. Market history, tracking price data on certain assets, trade volume etc. It's not there now but I thought it would be best to add the analytics tag as it'd be implemented pretty soon.

**Review by tweakch — CHANGES_REQUESTED** [review]
I suggest you remove the `analytics` tag. This could be seen as misleading

**Review by MarcelKlammer — APPROVED** [review]

**Review by Mercurial — APPROVED** [review]

**Review by katomm — CHANGES_REQUESTED** [review]
Sorry but the logo/screenshot still claims "Cardano's first CNFT Marketplace". 

Please also note that if people visit the site through the link they will receive a certificate error, because you link to www. but the certificate seems to be only for the domain without www.

**Comment by katomm** [comment]
Closing due to inactivity [15 days] (If you think this is a mistake, the PR can also be reopened)

**Pull request closed without merging**